src/lib.rs

```diff
@@ -11,7 +11,7 @@ use crate::{
     },
 };
 use anyhow::{bail, Result};
-use providers::Provider;
+use providers::{haskell::HaskellStackProvider, Provider};
```

Comment:
nit: Add this import to the above list (similar to how other providers are imported)

Reply:
Fixed!